android/app/build.gradle

```diff
@@ -38,7 +38,7 @@ android {
 
     defaultConfig {
         // TODO: Specify your own unique Application ID (https://developer.android.com/studio/build/application-id.html).
-        applicationId "com.example.eliana_chatbot"
+        applicationId "com.example.ui_trial"
```

Comment:
This shouldn't have changed

Reply:
My bad I cloned and pasted my files, thinking that it would be better to push a completely working repo from my side. Will make the necessary changes.